src/ztc/datamodel/models/resultaattype.py

```diff
@@ -6,8 +6,10 @@
 import requests
 from zds_schema.constants import (
     Archiefnominatie,
-    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze, ObjectTypes
+    BrondatumArchiefprocedureAfleidingswijze as Afleidingswijze,
+    ZaakobjectTypes
 )
+
```

Comment:
this blank line shouldn't be here, according to `isort`

Reply:
fixed it